pkg/manifest/testdata/fastly-viceroy-update.toml

```diff
@@ -45,3 +45,14 @@ qux"""
     [[local_server.object_store.store_two]]
       key = "second"
       path = "strings.json"
+
+  [local_server.secret_store]
+    store_one = [{key = "first", data = "This is some secret data"}, {key = "second", path = "/path/to/secret.json"}]
+
+    [[local_server.secret_store.store_two]]
+      key = "first"
+      data = "This is also some secret data"
+
+    [[local_server.secret_store.store_two]]
+      key = "second"
+      path = "/path/to/other/secret.json"
```

Comment:
This file is also at `pkg/commands/compute/testdata/init/fastly-viceroy-update.toml` so it should be copied there as well.

Reply:
Thanks, added in 662fa1c